.gitignore

```
@@ -74,3 +74,6 @@ google-credentials.json
 
 # vscode
 .vscode/
+
+# mac
+.DS_Store
```

```suggestion
**/.DS_Store
```
This only removes the .DS_Store in the root directory